bot/bot.py

```diff
@@ -135,6 +137,7 @@ def _recreate(self) -> None:
 
         # Create the redis session
         self.loop.create_task(self._create_redis_session())
+        self.redis_closed = False
```

Comment:
Small comment, this runs before `_create_redis_session` starts. The previous line just schedules that for execution later in the event loop.

Reply:
hm, good point. could move it into the `_create_redis_session` itself